downstream/titles/hub/configuring-private-hub-rh-certified/master.adoc

```diff
@@ -1,33 +1,44 @@
+
 :imagesdir: images
 = Managing Red Hat Ansible Content Collections and Ansible Galaxy collections in automation hub
 :numbered:
 :experimental:
 
 include::attributes/attributes.adoc[]
 
-You can sync {HubName} to use the Red Hat Certified Collections available through your {PlatformNameShort} subscription or community collections available through {Galaxy}.
+Included with a subscription to {PlatformName}, {CertifiedColl} feature {CertifiedName} and {Valid}. 
+Using {HubNameMain}, your organization can access and curate a unique set of collections from all forms of Ansible content. 
+
+You can also synchronize {HubNameMain} to create a custom list of collections from either {CertifiedCon}, {valid}, or {Galaxy} community content. 
 
-Your organization can access and curate into a unique set of collections from all Red Hat Certified content in the {HubName} service hosted on console.redhat.com.
+At present, Ansible validated collections are only available in your {PrivateHubName} {Valid} as part of the Platform Installer.
+When you download the bundled installer by default Validated Content is pre-populated into the {PrivateHubName}, if the {PrivateHubName} is enabled as part of the inventory.
```

Comment:
I think a comma should be added before "by default".

When you download the bundled installer, by default ...